server/models/bankoperation.coffee

```diff
@@ -8,6 +8,7 @@ module.exports = BankOperation = americano.getModel 'bankoperation',
     raw: String
     dateImport: Date
     categoryId: String
+    binary: (x) -> x
```

Comment:
IIUC cozy-data-system's code, binary is an object but it doesn't contain the file itself. Is that correct? If so, a comment would be appreciated here (for instance, an example of what a binary object could be).